examples/api-gateway/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN echo ' \
 {\
   "name": "package-base", \
   "dependencies": { \
-    "ocaml": "~4.7.1000", \
+    "ocaml": "*", \
```

Comment:
We want this to be the latest version that we support, so it should be synced with whatever is in `esy.json` (`4.9.0` at the moment)

Reply:
Done.